latest_v9_release.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "location": "https://download.katalon.com/7.8.2/Katalon_Studio_Windows_32-7.8.2.zip",
+        "file": "win_32",
+        "type": "kse"
+    },
+    {
+        "location": "https://download.katalon.com/9.0.0.beta/Katalon_Studio_Windows_64-9.0.0.beta.zip",
+        "file": "win_64",
+        "type": "kse"
+    },
+    {
+        "location": "https://download.katalon.com/9.0.0.beta/Katalon%20Studio.dmg",
+        "file": "mac_64",
+        "type": "kse"
+    },
+    {
+        "location": "https://download.katalon.com/9.0.0.beta/Katalon%20Studio%20Arm64.dmg",
+        "file": "mac_64",
```

Comment:
In 9.0.0 we have Mac ARM support so we should change the value here accordingly
Value is: mac_aarch64